### PR TITLE
Added Health Care Center functionalities

### PR DESCRIPTION
**Pull request by spectre900**
Added intent and response for health care center.

### CODE REVIEW

**Comment by adityasohoni** [comment]
Seems good. Just add a few more examples in the data. Also since the user may not know where the HCC is add a more appropriate response.

**Comment by spectre900** [comment]
added examples, changed the bot response, corrected typo and resolved conflicts

**Comment by adityasohoni** [comment]
Good work.
You can take up one of the other 'good first issues' now or add another functionality.
If you have any other suggestions you can discuss them too.

File: data/stories.yml [hunk 1]
```diff
@@ -118,6 +118,22 @@ stories:
...
+  - intent: Health_Care_Centre
+  - action: utter_go_to_health_care_Centre
+  - action: utter_did_that_help
+  - intent: affirm
+  - action: utter_happy
+
+- story: sad Health_Care_Centre
+  steps:
+  - intent: Health_Care_Centre
+  - action: utter_go_to_health_care_Centre
```
Comment (dharmicksai) [inline data/stories.yml:132]: hey here action name is wrong u have defined utter_go_to_health_care_centre in domain but here u have utter_go_to_health_care_Centre

Comment (spectre900) [reply data/stories.yml:132]: I didn't notice as it seemed to be working fine, it's probably case insensitive. I will fix this in the next PR.

Comment (dharmicksai) [reply data/stories.yml:132]: if u try training the bot u will get an error

Comment (spectre900) [reply data/stories.yml:132]: I remember now ,I had used "replaceAll" in this file for correcting a typo. Forgot to retrain and test. Will fix this in the next PR.

**Pull request merged**